mobile/lib/shared/services/event.service.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:event/event.dart';
+
+final scrollToDateEvent = Event<Value<DateTime>>();
```

Comment:
We do not need a separate third party package for this functionality. The same can be achieved using `ValueListenable` or even `riverpod`. We can implement a event system over this to notify widgets about updates. Can you please remove the third party library and use a different implementation instead?